### DISCUSSION TITLE
Making new releases with the latest shadowsocks-rust

### DISCUSSION THREAD

**Discussion by zonyitoo** [shadowsocks/shadowsocks-android#2865] [Ideas]
shadowsocks-rust recently released v1.14.1. It have been included many improvements since 2021-09-17 (v5.2.6 release date).

**Comment by dev4u** [comment]
其实针对手机的功能，并没有太大刚需做升级。因为你没有安卓手机，做的功能主要是针对pc上的ᥬ😂᭄

**Comment by dev4u** [comment]
#2887 等待code review
